p2p/protocol/circuitv2/relay/options.go

```diff
@@ -18,6 +18,14 @@ func WithLimit(limit *RelayLimit) Option {
 	}
 }
 
+// WithInfiniteLimits is a Relay option that disables limits.
+func WithInfiniteLimits() Option {
```

Comment:
```suggestion
// WithInfiniteLimits is a Relay option that disables limits.
// Enabling this option doesn't apply any filter for which protocols can be used.
func WithInfiniteLimits() Option {
```

Would it be valuable to add this additional comment?

Reply:
It's not the relay that limits the protocols, this is entirely a decision that the two endpoints using the relay make.